.circleci/config.yml

```diff
@@ -24,6 +25,13 @@ jobs:
             - flux-gomod-{{ .Branch }}-{{ .Revision }} # Matches when retrying a single run.
             - flux-gomod-{{ .Branch }}-                # Matches a new commit on an existing branch.
             - flux-gomod-                              # Matches a new branch.
+      # Populate Rust cache
```

Comment:
I guess this is like the GOCACHE but for Rust compilation artifacts?

Reply:
Yes, and we use a tool called `sccache` which allows you to specify a size limit on the cache as an LRU, otherwise Rust will grow the cache indefinitely.